measurement_octree_filename: /home/titoirfan/etc/soca_octomap_ws/src/soca_octomap/soca_octomap/example/octomaps/meas_octomap.bt
ground_truth_octree_filename: /home/titoirfan/etc/soca_octomap_ws/src/soca_octomap/soca_octomap/example/octomaps/ref_octomap.bt